checker/check_request.go

```diff
@@ -31,5 +31,6 @@ type CheckRequest struct {
 	HTTPClient  *http.Client
 	RepoClient  clients.RepoClient
 	Logf        func(s string, f ...interface{})
```

Comment:
`Logf` will ultimately be removed.

Reply:
Super nit: instead of review comments will help if you added these as code comments :)

Reply:
agreed. I pushed the PR and then realized it needed some explanation. Thanks for reminding me of adding the comments in the code!